tmt.spec

```diff
@@ -74,7 +74,7 @@ Dependencies required to run tests in a container environment.
 Summary: Virtual machine provisioner for the Test Management Tool
 Obsoletes: tmt-testcloud < 0.17
 Requires: tmt == %{version}-%{release}
-Requires: python%{python3_pkgversion}-testcloud >= 0.5.0
+Requires: python%{python3_pkgversion}-testcloud >= 0.6.0
```

Comment:
It is also necessary to update `setup.py` (I ran into exceptions due to having 0.5.0 and was wondering why it wouldn't update through pip ;)). I will create a PR for that. I've also ran into some problems with testcloud 0.6.0 and it does not work on my machine.